### ISSUE TITLE
Callback events do not work properly in UnityEditor after updating MAX Plugin to 5.0.0.

### ISSUE THREAD

**Issue by takazawa-gg** [AppLovin/AppLovin-MAX-Unity-Plugin#116]
After updating MaxPlugin to 5.0.0 with DemoApp, the following error will be output.

```
Error [AppLovin MAX] Failed to forward event for serialized event data: name=OnInterstitialLoadedEvent
adUnitId=ENTER_INTERSTITIAL_AD_UNIT_ID_HERE
UnityEngine.Debug:LogError (object)
MaxSdkLogger:E (string) (at Assets/MaxSdk/Scripts/MaxSdkLogger.cs:67)
MaxSdkCallbacks:ForwardEvent (string) (at Assets/MaxSdk/Scripts/MaxSdkCallbacks.cs:1111)
MaxSdkUnityEditor/<>c__DisplayClass65_0:<LoadInterstitial>b__0 () (at Assets/MaxSdk/Scripts/MaxSdkUnityEditor.cs:693)
MaxSdkUnityEditor/<ExecuteAction>d__104:MoveNext () (at Assets/MaxSdk/Scripts/MaxSdkUnityEditor.cs:1209)
UnityEngine.SetupCoroutine:InvokeMoveNext (System.Collections.IEnumerator,intptr)
```

Change the event string to JSON format and it will work.

MaxSdkUnityEditor.cs:693

Before:
```
MaxSdkCallbacks.Instance.ForwardEvent("name=OnInterstitialLoadedEvent\nadUnitId=" + adUnitIdentifier);
```

After:
```
MaxSdkCallbacks.Instance.ForwardEvent("{ \"name\": \"OnInterstitialLoadedEvent\", \"adUnitId\": \"" + adUnitIdentifier + "\" }");
```

Unity version: 2019.4.23f1
AppLovin Max plugin version: 5.0.0

**Comment by santoshbagadi** [comment]
Hi @takazawa-gg , thank you for reporting this! This was an oversight on my part. We will fix this in the next release, and thank you for adding the workaround in the mean time.

**Comment by LagField** [comment]
because of this issue,5.0.0 is not a useable version

**Comment by santoshbagadi** [comment]
Hi @LagField , @takazawa-gg , we have just released version [5.0.1](https://github.com/AppLovin/AppLovin-MAX-Unity-Plugin/releases/tag/release_5_0_1) of our plugin that fixes this issue. Please let us know if you still run into any issues after upgrading to this version.
